menus/views.py

```diff
@@ -0,0 +1,103 @@
+from django.shortcuts import render, get_object_or_404
+from django.utils import timezone
+from django.http import HttpResponse
+from django.views import generic
+from django.forms import formset_factory
+from django.http import HttpResponseRedirect
+
+from .models import Menu, Item
+
+from .forms import CreateForm, CreateFormSet, RestaurantForm
+
+class IndexView(generic.ListView):
+	template_name = 'menus/index.html'
+	context_object_name = 'menu_list'
+
+	def get_queryset(self):
+		return Menu.objects.order_by('-pub_date')
+
+class CreateView(generic.CreateView):
+	model = Item
+	success_url = '/'
+	template_name = 'menus/create.html'
+	form_class = CreateForm
+
+	def get_context_data(self, **kwargs):
+		print(kwargs)
+		context = super(CreateView, self).get_context_data(**kwargs)
+		context['formset'] = CreateFormSet(queryset=Item.objects.none())
+		context['restaurant_form'] = RestaurantForm()
+		return context
+
+	def post(self, request, *args, **kwargs):
+		formset = CreateFormSet(request.POST)
+		restaurant_form = RestaurantForm(data=request.POST)
+		if formset.is_valid() and restaurant_form.is_valid():
+			return self.form_valid(formset,restaurant_form)
+		else:
+			return render(request, 'menus/create.html', {
+				'formset': formset, 
+				'restaurant_form': restaurant_form,
+				'error_message': 'Invalid Input'})
```

Comment:
can you use a Flash Message here instead of this error message?
https://docs.djangoproject.com/en/3.0/ref/contrib/messages/